flytekit/deck/renderer.py

```diff
@@ -14,17 +14,22 @@ def to_html(self, python_value: Any) -> str:
         raise NotImplementedError
 
 
+DEFAULT_MAX_ROWS = 10
+DEFAULT_MAX_COLS = 100
+
+
 class TopFrameRenderer:
     """
     Render a DataFrame as an HTML table.
     """
 
-    def __init__(self, max_rows: Optional[int] = None):
+    def __init__(self, max_rows: Optional[int] = DEFAULT_MAX_ROWS, max_cols: Optional[int] = DEFAULT_MAX_COLS):
```

Comment:
not optional if you give a default.
```suggestion
    def __init__(self, max_rows: int = DEFAULT_MAX_ROWS, max_cols: int = DEFAULT_MAX_COLS):
```

Reply:
right?  or not?  i never figured out how this works?  i think it's from the perspective of the body of the function right?  not the caller

Reply:
that's true, we only need `Optional` if we want to explicitly support passing `None`, which is not the case here.